Sources/ApolloCodegenLib/OptionalBoolean.swift

```diff
@@ -1,8 +1,7 @@
 import Foundation
 
 extension Optional where Wrapped == Bool {
-  
-  var apollo_boolValue: Bool {
+  var valueOrFalseIfNone: Bool {
```

Comment:
I'd still like to keep this as `boolValue` - ultimately it behaves similarly to `[NSNumber boolValue]` in that if you call it on `nil` it returns false.

Reply:
OK. I changed it to `boolValue` with a comment on it.